synapse/config/oidc_config.py

```diff
@@ -158,6 +161,10 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
           #
           #skip_verification: true
 
+          # if user already exists, add oidc token to that account instead of failing. Defaults to false.
+          #
+          #merge_with_existing_users: false
```

Comment:
this doesn't seem a great name for this option. How about `allow_existing_users` ?